docs/tutorials.rst

```diff
@@ -0,0 +1,55 @@
+Tutorials
+==============
+
+The CAP form
+----------------
+There are four main section to document a physics analysis on the CAP form, they are basic information, data provenance, analysis software and documentation. However, each form is tailored to the needs of its experiment. In the following examples we use form snippets from different experiments to demonstrate their flexibility and use.
```

Comment:
Proposed changes:
There are four main sections to document a physics analysis on the CAP form: basic information, data provenance, analysis software and documentation. However, each form is tailored to the needs of each experiment. In the following examples, we use form snippets from different experiments to demonstrate their flexibility and use.

Reply:
👍